addons/account/views/account_move_views.xml

```diff
@@ -1460,7 +1459,7 @@
                     <separator/>
                     <filter string="Reversed" name="reversed" domain="[('payment_state', '=', 'reversed')]"/>
                     <separator/>
-                    <filter string="To Check" name="to_check" domain="[('to_check', '=', True)]"/>
+                    <filter string="To Check" name="checked" domain="[('checked', '=', False), ('state', '!=', 'draft')]"/>
```

Comment:
```suggestion
                    <filter string="To Check" name="to_checked" domain="[('checked', '=', False), ('state', '!=', 'draft')]"/>
```
Makes more sense to keep to_check I guess, since the domain is checked=False